src/exceptions/UnsupportedControlModeException.java

```diff
@@ -0,0 +1,8 @@
+package exceptions;
+
+public class UnsupportedControlModeException extends Exception {
```

Comment:
Use [unchecked exceptions](https://howtodoinjava.com/java/exception-handling/checked-vs-unchecked-exceptions-in-java/)